test_data/std/split_bn254_test.asm

```diff
@@ -0,0 +1,88 @@
+use std::split::split_bn254::SplitBN254;
+
+
+machine Main {
+    reg pc[@pc];
+    reg X0[<=];
+    reg X1[<=];
+    reg X2[<=];
+    reg X3[<=];
+    reg X4[<=];
+    reg X5[<=];
+    reg X6[<=];
+    reg X7[<=];
+    reg X8[<=];
+    reg A1;
+    reg A2;
+    reg A3;
+    reg A4;
+    reg A5;
+    reg A6;
+    reg A7;
+    reg A8;
+
+    degree 65536;
```

Comment:
I think we shouldn't have degree statements in std machines, because if a program with a longer ROM uses this, the compiler issues an error and the user needs to manually change it. Maybe we should have something like a  "lower bound" thing, because this is what it is, right?

Reply:
Ah wait, this is a test file so it's fine

Reply:
I agree, but this is just the test!